src/app/modules-table/modules-table.component.html

```diff
@@ -0,0 +1,79 @@
+<div class="mat-elevation-z8 module-table pt-20">
+  <table class="table" mat-table [dataSource]="dataSource">
```

Comment:
rename class `table` to any other name as table is tag name. may be `module-table`